### PR TITLE
Add support for Ubuntu Jammy (22.04)

### PR DESCRIPTION
**Pull request by kajinamit**
This change introduces initial support for Ubuntu Jammy.

Fixes: #227

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -16,7 +16,10 @@
       $user               = 'bind'
       $group              = 'bind'
       $rndcconfgen        = '/usr/sbin/rndc-confgen'
-      $named_checkconf    = '/usr/sbin/named-checkconf'
+      $named_checkconf    = $facts['os']['name'] ? {
+        'Ubuntu' => if versioncmp($facts['os']['release']['major'], '22.04') >= 0 { '/usr/bin/named-checkconf' } else { '/usr/sbin/named-checkconf' }
+        default  => '/usr/sbin/named-checkconf'
```
Comment (ekohl) [inline manifests/params.pp:21]: I think this fails on missing commas:
```suggestion
        'Ubuntu' => if versioncmp($facts['os']['release']['major'], '22.04') >= 0 { '/usr/bin/named-checkconf' } else { '/usr/sbin/named-checkconf' },
        default  => '/usr/sbin/named-checkconf',
```

**Review by ekohl — CHANGES_REQUESTED** [review]

File: manifests/params.pp (hunk 1 above)
Comment (kajinamit) [reply manifests/params.pp]: Oops. Fixed these.

**Comment by kajinamit** [comment]
@ekohl Thanks for merging this PR. Do you mind creating a new release ?

**Comment by ekohl** [comment]
https://github.com/theforeman/puppet-dns/pull/229

**Pull request merged**